### PR TITLE
[msal-common] V1 release

### PR DESCRIPTION
**Pull request by pkanher617**

### CODE REVIEW

File: lib/msal-browser/src/config/Configuration.ts [hunk 1]
```diff
@@ -29,16 +40,17 @@ export type BrowserAuthOptions = {
...
 };
 
 /**
  * Library Specific Options
  *
- * - logger                       - Used to initialize the Logger object; TODO: Expand on logger details or link to the documentation on logger
+ * - tokenRenewalOffsetSeconds    - Sets the window of offset needed to renew the token before expiry
+ * - loggerOptions                - Used to initialize the Logger object (See ClientConfiguration.ts)
+ * - networkClient                - Network interface implementation
+ * - windowHashTimeout            - sets the timeout for waiting for a response hash in a popup
```
Comment (jo-arroyo) [inline lib/msal-browser/src/config/Configuration.ts:51]:
```suggestion
 * - windowHashTimeout            - Sets the timeout for waiting for a response hash in a popup
```

File: lib/msal-browser/src/config/Configuration.ts [hunk 2]
```diff
@@ -29,16 +40,17 @@ export type BrowserAuthOptions = {
...
+ * - iframeHashTimeout            - sets the timeout for waiting for a response hash in an iframe
```
Comment (jo-arroyo) [inline lib/msal-browser/src/config/Configuration.ts:52]:
```suggestion
 * - iframeHashTimeout            - Sets the timeout for waiting for a response hash in an iframe
```

File: lib/msal-browser/src/config/Configuration.ts (hunk 1 above)
Comment (pkanher617) [reply lib/msal-browser/src/config/Configuration.ts:51]: Will update this in a following PR.

File: lib/msal-browser/src/config/Configuration.ts (hunk 2 above)
Comment (pkanher617) [reply lib/msal-browser/src/config/Configuration.ts:52]: Will update this in a following PR.

**Pull request merged**